src/ontology/mondo-edit.obo

```diff
@@ -450622,6 +450623,23 @@ intersection_of: MONDO:0100308 ! atactic disorder
 intersection_of: disease_has_basis_in_dysfunction_of UBERON:0004681 ! vestibular system
 created_by: https://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100313
+name: nephrogenetic disease
+def: "A type of kidney disorder that is characterized Mendelian (autosomal dominant, recessive, or X-linked) or chromosomal etiology and by morphological or functional abnormalities in the kidney or urinary system that dominate the clinical picture." []
+is_a: MONDO:0005240 ! kidney disorder
+property_value: http://purl.org/dc/elements/1.1/date "2023-04-01T19:18:35Z" xsd:dateTime
+created_by: peter
+
+[Term]
```

Comment:
@pnrobinson I don't think we need this term, MONDO:0005363 'focal segmental glomerulosclerosis' has a superclass assertion: 'bearer of' some inherited, so it already is the genetic form.

Reply:
ok! We can do that. :)